marvin-ai/marvin-python-toolbox
# Tk docker

Added support to put models in production using docker, the simplest way. This fixes issue #35

To test:

- create a new engine from scratch
- call dryrun
- call make docker-build
- call make docker-run
- send a pipeline request and check log to see if any error shows up
- finally call the make docker-push to send container image to repository and here is necessary one valid repository url
